### PR TITLE
Contributing guide

### PR DESCRIPTION
**Pull request by Superbelko**
Added some kind of tutorial on how to get started with contributing.

### CODE REVIEW

File: 001_Introduction.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- has some D experience
+- understanding OOP paradigm
+- is at least familiar with JSON file format structure
+- has basic proficiency level in command-line terminal
+
+## What to expect
+
+Before diving in here is an overview of what to expect. Many of the steps described in these recipes is now automated, however as an advanced users or contributors these steps allows to better understand underlying tech.
+
+Note that we start with we will work on a cloned repo instead of dub package because this guide is written for potential contributors.
```
Comment (nxuv) [inline 001_Introduction.md]: "that we start with we will" sound really wierd

File: 003_Basic-Example.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+![reparentnode.png](media/ch003/reparentnode.png)
+
+Move the `Rotator` node a bit away from top left corner, and reset `Icon` transform offset so it will rotate around its center and not some arbitrary point on scene.
+
+Don't forget to save the scene.
+
+Hit play button, and...
+
+Holy sh...
```
Comment (nxuv) [inline 003_Basic-Example.md]: Very professional :>

**Review by nxuv — APPROVED** [review]
Everything's fine except some often word repeats, I'd personally do more "professional" style, but it's ok as is except for "oh shi.." part (really not digging it)
Great job, guide seems to be easy to understand even for non-powerusers

**Comment by nxuv** [comment]
You can merge it now or fix it and merge after

File: 001_Introduction.md (hunk 1 above)
Comment (Superbelko) [reply 001_Introduction.md]: Oops, lots of re-editing has left some traces.

File: 003_Basic-Example.md (hunk 2 above)
Comment (Superbelko) [reply 003_Basic-Example.md]: It has a nice touch of free spirit of open source movement O_-

We can leave it as is or if it sounds too rude a boring "Oh no" will do.

**Pull request merged**